pandunia-loge.csv

```diff
@@ -197,7 +196,7 @@ Atlanti Hai|||eng:Atlantic, spa:Atlántico, por:Atlântico, rus:Атлантич
 atlanti salmon|bio|Salmo salar||Atlantic salmon|saumon altentique|salmón del Atlántico||Атлантический лосось (сёмга)||大西洋鮭|タイセイヨウサケ|||||salmon Atlantik|||||łosoś atlantycki
 atom|||eng:tur:may:atom, spa:por:átomo, rus:атом (atom), swa:atomi, jpn:アトム (atomu)|atom|atome|átomo|||||||||||||atomo|atomi|atom
 audi|||eng:spa:fra:audio, por:áudio|hear (listen)||oír (escuchar)|||||||||||||aŭdi (aŭskulti)|kuulla (kuunnella)|usłyszeć, słyszeć; słuchać
-auro|mate|Au|ltn:aurum, por:ouro, spa:oro, fra:or, rus:аурум (aurum), may:aurum|gold|or|oro|ouro|золото (аурум)||金|金|금|vàng|सोना|সোনা|emas (mas, aurum)|dhahabu|altın|oro|kulta|złoto
+auro|yum 079|Au|ltn:aurum, por:ouro, spa:oro, fra:or, rus:аурум (aurum), may:aurum, swa:auri|gold|or|oro|ouro|золото|ذهب|金|金|금|vàng|सोना|গোল্ড|emas|dhahabu (auri)|altın|oro|kulta|złoto
```

Comment:
I noticed that gold is **auro** and iron is **feru**.  but they should both end in the same vowel, rite?

Reply:
Makes sense! Both shall end in -o.